routes/trigger/trigger_api.go

```diff
@@ -43,6 +45,22 @@ func New(controller decomposer_controller.IController, database database.IContro
 	return &instance
 }
 
+func MergeVariablesInPlaybook(playbook *cacao.Playbook, body []byte) {
+
+	variables := cacao.NewVariables()
+	err := json.Unmarshal(body, &variables)
+	if err != nil {
+		log.Trace(err)
+		return
+	}
+
+	if playbook.PlaybookVariables == nil {
+		playbook.PlaybookVariables = variables
```

Comment:
Does this initialise a nil playbookVariables object? In that case, we might want to initialise an *empty* playbookVariables object instead, as per recent merge request - right?

Reply:
I stand corrected, variables, is assigned. All good.